### DISCUSSION TITLE
Dates from table are a day behind

### DISCUSSION THREAD

**Discussion by jacobmaussocc** [Budibase/budibase#11336] [Help] [answered]
Hi, I am pulling data from tables and the date on the tables on the MySQL database is all on Fridays, when I pull it into Budibase it makes it all Thursdays. It is like it is a day behind. I really want to understand a fix. Thanks. (Ex. 07/21/2023 turns into 07/20/2023)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @jacobmaussocc,

This is likely related to your timezone, have you tried using the ignore timezones check box on the field, also make sure its checked on the data fields column within your data section.

<img width="313" alt="Screenshot 2023-07-25 at 07 51 32" src="https://github.com/Budibase/budibase/assets/126772285/80459f8b-cf82-4f1b-b216-bb2f68b04750">
<img width="402" alt="Screenshot 2023-07-25 at 07 52 52" src="https://github.com/Budibase/budibase/assets/126772285/46e20e54-e2a4-4c2d-973b-fa5d55422a68">

If neither of the above works to resolve your issue, there will be a new date field added soon that won't include time which will indirectly fix this issue for you. (There isn't a public issue for this at the moment, however this is currently being worked on and should be added to the core product over the coming months.)

I hope this is helpful.

**Reply by jacobmaussocc** [reply]
Thanks for the help! While we are here. Is there a way in Budibase where I can change the date format I retrieve from a query? Right now the date when I query is "2023-07-21", I would prefer "Jul 21, 2023" or something along those lines.

**Reply by ConorWebb96** [reply]
Hey @jacobmaussocc,

You should be able to switch to the JS within the area you are outputting the date and use a a function to manipulate the date into whatever format you desire. A function like this in the bindings area should work for you;

```
  const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
  const dateObj = new Date({{ dateValueHere }});
  const formattedDate = `${months[dateObj.getMonth()]} ${dateObj.getDate()}, ${dateObj.getFullYear()}`;
  return formattedDate;
```

I hope this helps!
